crf/src/main/assets/json/heart_snapshot.json

```diff
@@ -38,10 +38,10 @@
     {
       "identifier": "sexForm",
       "type": "crf_form",
-      "title": "What is your sex?",
-      "learnMoreText"    : "Why are we asking about this?",
+      "title": "What is your gender?",
```

Comment:
@nategbrown9 also fyi there was a push on iOS to change all instances of "sex" to "gender" so I went and matched all the copy that iOS has for this. I kept the back-end identifier as "sex"; however, as changing that was out of scope and iOS still has "sex" as the identifier as well.

Reply:
Interesting. I thought the HR recovery formulas we use use sex at birth (the studies were done decades ago)

Reply:
It's a good thought, I'm going to have Dan write up a few sentences on motive/why it's OK to make the change from "sex" to "gender" scientifically to the Jira issue, just so we have it documented.

Reply:
Studies performed decades ago would not have asked for a participant's "sex at birth" and the way they likely determined sex (probably a line on a questionnaire "Sex (m/f):") wouldn't even make a distinction between sex/sex assigned at birth/gender so it's moot anyway--they wouldn't have had the data to determine which is actually most relevant to the calculation. But documentation is good.

Reply:
Good to know. I had thought they used what is on your birth certificate back then. Hopefully we'll start seeing **many** more studies to establish baseline data for all sorts of common biomarkers for populations that historically weren't adequately represented

Reply:
Promising implications for improvement in outcomes. I've followed the BMI and waist measurement renorm studies for API and the downstream impacts on cardiovascular and diabetic risk assessment are significant.